tests/unit/amo/components/TestAddonTitle.js

```diff
@@ -26,54 +26,64 @@ describe(__filename, () => {
       addon: createInternalAddon({ ...fakeAddon, name }),
     });
 
-    expect(root.html()).toContain(name);
+    expect(root).toIncludeText(name);
   });
 
   it('renders a single author', () => {
-    const authorUrl = 'http://olympia.test/en-US/firefox/user/krupa/';
+    const author = {
+      ...fakeAddon.authors[0],
+      username: 'some-username',
```

Comment:
You don't technically need to specify a specific `username` in the test setup because you are never referring to this later. You just use `author.username` in your assertion, so it doesn't really matter what the `username` is. You don't specify an author name here, so why specify a `username`?